packages/ai-jsx/src/react/index.ts

```diff
@@ -188,4 +188,3 @@ export const jsx = asJsxBoundary(function jsx(
 
   return ReactModule.createElement(ReactModule.Fragment, null, waitingForFirstAIResponse ? loading : ai.result) as any;
 });
-export const JSX = jsx;
```

Comment:
bad merge? the purpose this serves is satisfying persnickety lint rules that prefer `<AI.JSX>` to `<AI.jsx`>